.github/pull_request_template.md

```diff
@@ -1,32 +1,36 @@
 ## Purpose
 
-* https://labrys.atlassian.net/browse/XXX-NNNN
-* Outline what the PR is addressing in a brief sentence form
-* Explain how it relates to the Jira card, if it completes it or what is left to complete it
+- https://labrys.atlassian.net/browse/XXX-NNNN
+- Outline what the PR is addressing in a brief sentence form
+- Explain how it relates to the Jira card, if it completes it or what is left to complete it
 
 ## Approach
 
 All these are optional depending on the size and context of the work being done.
 
-* A high-level account of how the solution was achieved
-* You might include some names of components/files for clarity, but is meant to be a general description
-* Highlight the starting point for the review, like the highest level component or page
-* Link to the automated test that proves the feature is working
-* Provide a sequence or class diagram of the areas affected, with accurate names of the components and functions involved. This is optional unless you generated it with an agent or created one for the work (smart contract designs for instance).
+- A high-level account of how the solution was achieved
+- You might include some names of components/files for clarity, but is meant to be a general description
+- Highlight the starting point for the review, like the highest level component or page
+- Link to the automated test that proves the feature is working
+- Provide a sequence or class diagram of the areas affected, with accurate names of the components and functions involved. This is optional unless you generated it with an agent or created one for the work (smart contract designs for instance).
 
 ## QA
 
-* Provide the paths at which the UI have changed
-* Instructions on how to navigate to the areas affected by the PR
-* Explain how it was before
-* Explain how these changes affect the UI/system
-* What should people notice after the changes have been introduced
+- Provide the paths at which the UI have changed
+- Instructions on how to navigate to the areas affected by the PR
+- Explain how it was before
+- Explain how these changes affect the UI/system
+- What should people notice after the changes have been introduced
 
 ## Key decisions
 
-* Important decisions with ramifications on future work
-* Tradeoffs
-* Heads up for new reusable components
+- Important decisions with ramifications on future work
+- Tradeoffs
+- Heads up for new reusable components
+
+## Breaking Changes
+
+<!-- List any breaking changes or write "None" -->
```

Comment:
If we want to use html comments for instructions, the whole file needs to be made consistent.

Reply:
Not sure what you mean by "all these files"?